onnx/defs/shape_inference.h

```diff
@@ -32,6 +32,18 @@ inline bool getRepeatedAttribute(InferenceContext& ctx,
 
 }
 
+inline bool hasExactlyNInputTypes(InferenceContext& ctx, int n, const std::string& opname) {
+  if (ctx.getNumInputs() != n) {
+    throw std::runtime_error(opname + " has wrong number of inputs");
```

Comment:
Why not return false, instead of throwing an exception? E.g., I see this being invoked for Concat, where this may not be an error.

Reply:
well, this is just a helper that throws an exception. the error is with concat - i'll remove that usage